src/frontend/js/widgets/Dashboard/components/DashboardAvatar/_styles.scss

```diff
@@ -11,16 +11,27 @@ $avatar-size: 100px;
     justify-content: center;
     align-items: center;
     color: r-theme-val(dashboard-avatar, base-color);
-    font-size: 3rem;
-    font-weight: bold;
+    padding: rem-calc(3px);
+
+    &__letter {
+      font-size: 2.5rem;
+      font-family: $r-font-family-montserrat;
+      font-weight: $font-weight-boldest;
+      line-height: 1;
+      position: absolute;
+      top: 50%;
+      left: 50%;
+      transform: translate(-50%, -50%);
+      text-transform: uppercase;
+    }
 
     &--square {
       border-radius: rem-calc(8px);
       background: r-theme-val(dashboard-sidebar, background-color);
       border: 1px solid r-theme-val(dashboard-avatar, background-color);
       color: r-theme-val(dashboard-avatar, background-color);
 
-      img {
+      & > .dashboard__avatar__image {
```

Comment:
We only have one avatar's image.
No need of `>` that would only select direct children.